src/pages/patientView/genomicOverview/Tracks.tsx

```diff
@@ -89,8 +89,8 @@ export default class Tracks extends React.Component<TracksPropTypes, {}> {
                 rowIndex = rowIndex + 1;
 
                 if (this.props.sampleManager.samples.length > 1) {
-                    const id = `#mutTrack${sample.id}`;
-                    const $container = $(id);
+                    const id = `mutTrack${sample.id}`;
+                    const $container = $(`[id="${id}"]`);
                     const pos = {x: parseInt($container.attr('x')) - 10, y: parseInt($container.attr('y')) - 5};
                     const $newContainer = $(`<svg id="${id}" height="12" width="12" />`);
```

Comment:
i kept the only other reference to id without `#` because that seems to have been a typo